profiles/common/modules/custom/nexteuropa_token/src/Entity/LinkTokenHandler.php

```diff
@@ -47,11 +47,12 @@ public function hookTokens($type, $tokens, array $data = array(), array $options
     $replacements = array();
 
     if ($this->isValidTokenType($type)) {
+      $token_types = token_get_entity_mapping('token');
```

Comment:
Function token_get_entity_mapping() by default provides mapping for the 'token' value;